invenio_app_ils/documents/loaders/jsonschemas/document.py

```diff
@@ -210,7 +210,7 @@ class Meta:
     identifiers = fields.List(fields.Nested(IdentifierSchema))
     imprint = fields.Nested(ImprintSchema)
     internal_notes = fields.List(fields.Nested(InternalNoteSchema))
-    keywords = fields.Nested(KeywordSchema)
+    keywords = fields.List(fields.Nested(KeywordSchema))
```

Comment:
Not sure about this change: `keywords` was supposed to be just a text field. Why a list?
We had a long discussion with the library to understand keywords and tags, if you remember.